resourcepolicies.md

```diff
@@ -10,34 +10,298 @@ As we don't have yet an use case to iterate over all the existent resource polic
+With the `remove` operation, you can delete:
+- startDate
+- endDate
+- name
+- description
+
+the other properties cannot be nullified.
+
+To remove an embargo you can for instance use the following
+`curl --data '[ { "op": "remove", "path": "/startDate" }]' -H "Authorization: Bearer ..." -H "content-type: application/json" -X PATCH ${dspace7-url}/api/authz/resourcepolicies/${resourcepolicy-id}`
+
+that will transform
+
+```json
+  "id": 2844,
+  "name": "my name",
+  "description": "my description",
+  "policyType": "TYPE_SUBMISSION",
+  "action": "READ",
+  "startDate" : "2019-10-31",
+  "endDate" : null,
+  "type": "resourcepolicy"
+```
+
+in 
+
+```json
+  "id": 2844,
+  "name": "my name",
+  "description": "my description",
+  "policyType": "TYPE_SUBMISSION",
+  "action": "READ",
+  "startDate" : null,
+  "endDate" : null,
+  "type": "resourcepolicy"
+```
+
+Use of the remove operation over a not acceptable path will result in a 422 Unprocessable Entity return code
+
+### Replace
+The replace operation allows to replace *existent* information with new one. Attempt to use the replace operation to set not yet initialized information must return an error. See [general errors on PATCH requests](patch.md)
+
+To change the startDate
+`curl --data '[ { "op": "replace", "path": "/startDate", "value": "2020-01-01" }]' -H "Authorization: Bearer ..." -H "content-type: application/json" -X PATCH ${dspace7-url}/api/authz/resourcepolicies/${resourcepolicy-id}`
+
+For example, starting with the following item data:
+
+```json
+  "id": 2844,
+  "name": null,
+  "description": null,
+  "policyType": "TYPE_SUBMISSION",
+  "action": "READ",
+  "startDate" : "2019-10-31",
+  "endDate" : null,
+  "type": "resourcepolicy"
+```
+
+the replace operation will result in:
+
+```json
+  "id": 2844,
+  "name": null,
+  "description": null,
+  "policyType": "TYPE_SUBMISSION",
+  "action": "READ",
+  "startDate" : "2020-01-01",
+  "endDate" : null,
+  "type": "resourcepolicy"
+```
+
+## Deleting a resource policy
+**DELETE /api/authz/resourcepolicies/<:id>**
+
+Return codes:
+* 204 No content - if the operation succeed
+* 401 Forbidden - if you are not authenticated
+* 403 Unauthorized - if you are not logged in with sufficient permissions. Only system administrators or user with ADMIN permission over the target resource can use the endpoint
+* 404 Not found - if the item doesn't exist (or was already deleted)
+
+
 ## Linked entities
 ### EPerson
 **/api/authz/resourcepolicies/<:id>/eperson**
 
-t.b.d
+Provide access to the eperson linked by this resource policy
+
+The recipient of the policy, eperson or group, cannot be modified. If you need to do so please delete the policy and create a new one.
+
 
 ### Group
 **/api/authz/resourcepolicies/<:id>/group**
 
-t.b.d
+Provide access to the group linked by this resource policy
+
+The recipient of the policy, eperson or group, cannot be modified. If you need to do so please delete the policy and create a new one.
```

Comment:
Can you add permission requirements and return codes (especially since it's nullable) for this
If regular users would be authorized to retrieve resource policies, can they also access all data about this object?

Reply:
done